src/App.jsx

```diff
@@ -14,57 +16,71 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const reset = () => {
+    return setSelectedGood(null);
```

Comment:
1. to avoid using two types of value you can just use an empty string instead
2. no needs to return a function
```suggestion
    setSelectedGood('');
```